### ISSUE TITLE
[Feature Request] start_level for coloring

### ISSUE THREAD

**Issue by themilkman** [absop/RainbowBrackets#3]
Hi @absop, thanks for this sweet plugin :-)

Do you think it'd be possible to add an "nesting_level"/"start_level" for the rainbows?
I just ask since generally like the common syntax highlighting as it is not disruptive etc. But on nested brackets this plugin of course shows its strength. So, I'd like to keep it off for the first level but would like to see it starting with the second.

Thanks for any feedback on this. Cheers!

**Comment by absop** [comment]
I have ever Designed this plugin have two modes to be options:
- scan and match all brackets, as it now in use,
- scan only part of text around the cursor, for a better efficiency.

But later I thought it sways eyes and of little use, so I remove it from the plugin.
I am not too clear about what functions you want, maybe you could download a old version to have a try, then give me more information.<https://github.com/absop/RainbowBrackets/tree/3.0.0>

**Comment by themilkman** [comment]
Thanks for your response, I'll give it a try in the next days :)

**Comment by themilkman** [comment]
Hi @absop,
sorry for the late response.
I tried the older version but didn't really notice the big difference.
But now, after thinking about it and playing around a bit, I simply came up with coloring the first bracket in `rainbow_colors` the same way as my color schema.
By doing this, I get less "color noise" in my files and only on more deeply nested cases it "starts" changing the color. This was my intention by the feature request.

Thanks again for your plugin as well as for your kind and super fast response!
